content/docs/apps/static-egress.md

```diff
@@ -5,15 +5,22 @@ menu:
 title: Communicating with external services
 ---
 
-If your cloud.gov-hosted application needs access to an external service, that service must be accessible over the internet. All traffic from cloud.gov applications will transit the internet.
+You can set up your cloud.gov application to communicate with any external service that is accessible over the internet. For example, you can integrate your application with services in your agency infrastructure or with APIs from commercial providers.
+
+All traffic from cloud.gov applications will transit the internet. You can secure the communication with authentication and with encryption in transit, and you can limit access to specific IP ranges.
 
 ## Securing communications with external services
 
 You can set up your cloud.gov application to communicate with any external service available over the internet. To ensure only your application can communicate with the external service, the service should be configured to authenticate each client using client-specific credentials.
 
+Authentication options:
+
+* **cloud.gov client application connecting to external services:** If your application on cloud.gov needs to be a client of an existing server application outside of cloud.gov, your application on cloud.gov can have a TLS certificate that the server application validates, or you can use application-level authentication (such as Authorization headers). In addition, you can configure outside services to accept connections from the [cloud.gov egress address ranges]((#cloud-gov-egress-ranges). You can accept HTTPS and TCP connections.
+* **cloud.gov server application accepting external client connections:** If you have a server application on cloud.gov that needs to secure connections from a client application outside of cloud.gov, your server application can limit inbound IP ranges using a [user-provided service instance](https://docs.cloudfoundry.org/devguide/services/user-provided.html) of a [route service](https://docs.cloudfoundry.org/devguide/services/route-binding.html), and you can set up application-level client authentication, such as Authorization headers. (*Limitations:* In the current cloud.gov architecture, your server application on cloud.gov cannot use protocol-level authentication, such as a TLS certificate, to authenticate with a client application outside of cloud.gov. Your server application can only provide HTTPS services, not TCP services.)
```

Comment:
> to authenticate with a client application outside of cloud.gov

I think it'd be better to say "to authenticate clients connecting to your server application".  "Authenticate with" makes it sound like the cloud-gov application is the client here when it's actually a server.